presto-main/src/test/java/com/facebook/presto/sql/TestExpressionInterpreter.java

```diff
@@ -1075,6 +1075,11 @@ public void testLikeOptimization()
     {
         assertOptimizedEquals("unbound_string like 'abc'", "unbound_string = CAST('abc' AS VARCHAR)");
 
+        assertOptimizedEquals("unbound_string like 'a\\_b' escape '\\'", "unbound_string = CAST('a_b' AS VARCHAR)");
+        assertOptimizedEquals("unbound_string like 'a\\%b' escape '\\'", "unbound_string = CAST('a%b' AS VARCHAR)");
+        assertOptimizedEquals("unbound_string like 'a\\_\\\\b' escape '\\'", "unbound_string = CAST('a_\\b' AS VARCHAR)");
```

Comment:
We should check the SQL standard to see how this is supposed to be handled. Ignore it? Raise an error?